### ISSUE TITLE
Show any errors from k8s endpoint

### ISSUE THREAD

**Issue by lijianzhi01** [kubernetes-sigs/headlamp#1643]
Edit or delete resources might have error code from k8s endpoint. It should be show error in UI. This is very easy for user to check issues.

**Comment by illume** [comment]
Yeah agreed. The next step is probably coming up with a UI for this.

A snackbar notification for deletion?

In the editor there's a validation error message that can be displayed down the bottom. I guess this would be the appropriate place for editor related error messages.

**Comment by joaquimrocha** [comment]
We are working on design to better implement this.
